src/core/status.h

```diff
@@ -27,6 +27,7 @@
 
 #include <string>
 #include "src/core/tritonserver_apis.h"
+#include "triton/common/status.h"
```

Comment:
It is confusing to have this Status and also common::Status. How about name the common class "Error". so it is triton::common::Error.